timeout/options.go

```diff
@@ -0,0 +1,65 @@
+package timeout
+
+import (
+	"fmt"
+	"time"
+
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+// CmdOptions Common Timeout Option.
+type CmdOptions struct {
```

Comment:
This can probably be named `Options` or `TimeoutOptions` unless I am missing something!

Reply:
a struct in a package timeout ist not supposed to be named anything starting with timeout. 
It is supposed to read timeout.CmdOptions. Instead of repeating timeout.TimeoutOptions. 

So Options would be fine. But,.. there was one place in the choose command Options, where this would clash as the Name Options was already used in the struct.
By the way, this struct also does not read to easily. 
A struct called Options containing a field Options. 

````
package choose

import (
	"github.com/charmbracelet/gum/style"
	common "github.com/charmbracelet/gum/timeout"
)

// Options is the customization options for the choose command.
type Options struct {
     Options []string `arg:"" optional:"" help:"CmdOptions to choose from."`

    common.CmdOptions              // including timeout command options [Timeout,...]
````

Reply:
IMHO The more golangey way would be to use this as a means of inheriting common probs w/o having the indirection through an extra property. But sure, for readability this is also an option.

Reply:
Renaming Options in the choose Options struct is not doable, as this would break existing user scripts

Reply:
just renamed and pushed

Reply:
Awesome! I agree, reads much better. Thank you so much for changing!